dashboard/app.py

```diff
@@ -18,6 +18,12 @@
 
 s3_client = boto3.client("s3")
 
+def get_formatted_times() -> list[datetime.datetime, datetime.datetime]:
+    TODAY = datetime.datetime.now()
+    TODAY_FORMATTED = datetime.datetime(TODAY.year, TODAY.month,
+                                        TODAY.day, TODAY.hour, TODAY.minute, TODAY.second)
+    LAST_DAY = TODAY_FORMATTED - datetime.timedelta(hours=12)
+    return [TODAY_FORMATTED, LAST_DAY]
```

Comment:
don't necessarily need to return as a list, could have just done return TODAY_FORMATTED, LAST_DAY